### DISCUSSION TITLE
How to change the default alias of a query and how to add queries in bulk when DataSource is mysql

### DISCUSSION THREAD

**Discussion by wypxl** [grafana/mimir#8158] [General]
Hello everyone,
My grafana's dataSource is mysql, it stored the power consumption per minute of over 200 cabinets, so I used 'Time series' to display them, now I have two questions that I hope you can help me with.
1.The name displayed of the data taken out on the panel is the column name of the power_consumption by default. How can I change this name to the cabinet_id searched by sql statement?
2.Normally, one query must be added to each cabinet. Can I add more than 200 cabinets in batches?

The data format is as follows, hope can get help from here, thanks a lot
![99fc369d2672938b229e7de616b8b8b](https://github.com/grafana/mimir/assets/45066702/b25403d4-ba11-4326-bc92-497aabeb9352)

**Comment by charleskorn** [comment]
This repository is specifically for Mimir, but your question seems to be about Grafana and querying MySQL. Please post your question in the Grafana community forum for help with Grafana: https://community.grafana.com/c/support/33.

**Reply by wypxl** [reply]
OK，thanks~
